cetmix_tower_server/wizards/cx_tower_command_execute_wizard_view.xml

```diff
@@ -52,10 +52,12 @@
                     <div class="o_row" attrs="{'invisible': [('result', '!=', False)]}">
 
                         <field name="action" nolabel="1" />
+                        <span
+                            attrs="{'invisible': [('action', '!=', 'ssh_command')]}"
+                        >sudo</span>
                         <field
                             name="use_sudo"
                             attrs="{'invisible': [('action', '=', 'python_code')]}"
-                            placeholder="No sudo"
```

Comment:
Not related to this PR, however would be reasonable to unify the visibility conditions with `<span>`
```
attrs="{'invisible': [('action', '!=', 'ssh_command')]}"
```